files/build_templates/sonic_debian_extension.j2

```diff
@@ -157,6 +157,10 @@ sudo sed -i -e '/^passwd/s/ tacplus//' $FILESYSTEM_ROOT/etc/nsswitch.conf
 # Copy crontabs
 sudo cp -f $IMAGE_CONFIGS/cron.d/* $FILESYSTEM_ROOT/etc/cron.d/
 
+## copy database instances checking script and config
+sudo cp $IMAGE_CONFIGS/database/ping_pong_single_db_inst.sh $FILESYSTEM_ROOT/usr/local/bin/
+sudo cp $IMAGE_CONFIGS/database/ping_pong_db_insts $FILESYSTEM_ROOT/usr/local/bin/
```

Comment:
>ping_pong_db_insts [](start = 32, length = 18)

Offline discussed and we agree move both script into database docker.

Reply:
done